src/Markup/Avalonia.Markup.Xaml/Avalonia.Markup.Xaml.csproj

```diff
@@ -8,58 +8,95 @@
+    <Compile Include="XamlIl\Runtime\IAvaloniaXamlIlXmlNamespaceInfoProviderV1.cs" />
+    <Compile Include="XamlIl\Runtime\XamlIlRuntimeHelpers.cs" />
+    <Compile Include="XamlLoadException.cs" />
+    <Compile Include="XamlTypes.cs" />
+    <Compile Include="AvaloniaXamlLoader.cs" />
+    <Compile Include="Converters\AvaloniaUriTypeConverter.cs" />
+    <Compile Include="Converters\ColorToBrushConverter.cs" />
+    <Compile Include="Converters\FontFamilyTypeConverter.cs" />
+    <Compile Include="Converters\TimeSpanTypeConverter.cs" />
+    <Compile Include="Extensions.cs" />
+    <Compile Include="MarkupExtension.cs" />
+    <Compile Include="MarkupExtensions\CompiledBindingExtension.cs" />
+    <Compile Include="MarkupExtensions\CompiledBindings\CompiledBindingPath.cs" />
+    <Compile Include="MarkupExtensions\CompiledBindings\PropertyInfoAccessorFactory.cs" />
+    <Compile Include="MarkupExtensions\CompiledBindings\TaskStreamPlugin.cs" />
+    <Compile Include="MarkupExtensions\DynamicResourceExtension.cs" />
+    <Compile Include="MarkupExtensions\On.cs" />
+    <Compile Include="MarkupExtensions\OnFormFactorExtension.cs" />
+    <Compile Include="MarkupExtensions\OnPlatformExtension.cs" />
+    <Compile Include="MarkupExtensions\ResolveByNameExtension.cs" />
+    <Compile Include="MarkupExtensions\StaticResourceExtension.cs" />
+    <Compile Include="Parsers\PropertyParser.cs" />
+    <Compile Include="Converters\BitmapTypeConverter.cs" />
+    <Compile Include="Converters\IconTypeConverter.cs" />
+    <Compile Include="Converters\AvaloniaPropertyTypeConverter.cs" />
+    <Compile Include="Converters\PointsListTypeConverter.cs" />
+    <Compile Include="MarkupExtensions\ReflectionBindingExtension.cs" />
+    <Compile Include="MarkupExtensions\RelativeSourceExtension.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="RuntimeXamlLoaderConfiguration.cs" />
+    <Compile Include="RuntimeXamlLoaderDocument.cs" />
+    <Compile Include="Styling\MergeResourceInclude.cs" />
+    <Compile Include="Styling\ResourceInclude.cs" />
+    <Compile Include="Styling\StyleInclude.cs" />
+    <Compile Include="Templates\ControlTemplate.cs" />
+    <Compile Include="Templates\DataTemplate.cs" />
+    <Compile Include="Templates\FocusAdornerTemplate.cs" />
+    <Compile Include="Templates\ItemsPanelTemplate.cs" />
+    <Compile Include="Templates\Template.cs" />
+    <Compile Include="Templates\TemplateContent.cs" />
+    <Compile Include="Templates\TreeDataTemplate.cs" />
+    <Compile Include="XamlIl\Runtime\IAvaloniaXamlIlControlTemplateProvider.cs" />
+    <Compile Include="XamlIl\Runtime\IAvaloniaXamlIlParentStackProvider.cs" />
+    <Compile Include="XamlIl\Runtime\IAvaloniaXamlIlXmlNamespaceInfoProviderV1.cs" />
+    <Compile Include="XamlIl\Runtime\XamlIlRuntimeHelpers.cs" />
+    <Compile Include="XamlLoadException.cs" />
+    <Compile Include="XamlTypes.cs" />
```

Comment:
I guess that it is incorrect that all files are included twice here now

Reply:
Must have been a merge error - well spotted! Fixed.